pom.xml

```diff
@@ -242,9 +242,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>2.9.1</version>
-                <configuration>
-                    <detectJavaApiLink>false</detectJavaApiLink>
-                </configuration>
```

Comment:
@sfc-gh-japatel did you add this intentionally in pom.xml but not pom_confluent.xml?

Reply:
It was few years ago but based on this: https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#detectJavaApiLink

I might have seen some warnings while building this repo. May be lets keep it as is or add it to confluent_pom?

Reply:
I don't see any related warning, you probably added as a workaround for some old issue, like https://issues.jenkins.io/browse/JENKINS-55692?src=confmacro. I feel like we should just use the default which is ture.